editor/gui/editor_file_dialog.cpp

```diff
@@ -1386,6 +1412,8 @@ void EditorFileDialog::_update_favorites() {
 
 	favorite->set_pressed(false);
 
+	// FIXME: `favorited` and `favorited_paths` seem to be the two same variables.
+	//        Not sure is this is intentional or not.
```

Comment:
They are not exactly the same, e.g. `favorited_names` uses `/` instead of `res://`.
Again, unnecessary comment 🙃